### PR TITLE
Update Pro Engine toggle

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/semgrep-code/semgrep-pro-engine-intro.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ To enable Semgrep Pro Engine in the Semgrep Cloud Platform, follow these steps:
 
 1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login).
 1. Select **[Settings](https://semgrep.dev/orgs/-/settings)**.
-1. Enable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle.
+1. Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine beta** toggle is enabled.
```
Comment (armchairlinguist) [inline docs/semgrep-code/semgrep-pro-engine-intro.md:139]:
```suggestion
1. Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine** toggle is enabled.
```

File: docs/semgrep-code/semgrep-pro-engine-examples.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Semgrep matches `dangerous(“Select * from “ + user_input)`, because `user_in
 Semgrep Pro Engine matches both dangerous calls because it does cross function boundaries. In fact, with Semgrep Pro Engine, the taint rule can track calls to `get_user_input` over multiple jumps in multiple files.
 
 :::tip Try it out
-Enable the **Semgrep Pro Engine beta** <i class="fa-solid fa-toggle-large-on"></i> toggle in the following link to an [example of dangerous taint](https://semgrep.dev/playground/s/J0dQ). To run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/returntocorp/semgrep-pro-tests). Go to `docs/taint_tracking/java` and run the following command:
+Ensure that the **Pro Engine beta** <i class="fa-solid fa-toggle-large-on"></i> toggle is enabled in the following link to an [example of dangerous taint](https://semgrep.dev/playground/s/J0dQ) rule. To run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/returntocorp/semgrep-pro-tests). Go to `docs/taint_tracking/java` and run the following command:
```
Comment (armchairlinguist) [inline docs/semgrep-code/semgrep-pro-engine-examples.md:48]:
```suggestion
Ensure that the **Pro Engine** <i class="fa-solid fa-toggle-large-on"></i> toggle is enabled in the following link to an [example of dangerous taint](https://semgrep.dev/playground/s/J0dQ) rule. To run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/returntocorp/semgrep-pro-tests). Go to `docs/taint_tracking/java` and run the following command:
```

File: docs/semgrep-code/semgrep-pro-engine-examples.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ This document provides an overview of Semgrep Pro Engine features through specif
 
 The following resources can help you to test the code in the sections below. As you work through the examples in this document, try the following:
 
-- Enable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle within the [Playground](https://semgrep.dev/playground/new).
+- Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine beta** toggle is enabled on the [Playground](https://semgrep.dev/playground/new) page.
```
Comment (armchairlinguist) [inline docs/semgrep-code/semgrep-pro-engine-examples.md:16]:
```suggestion
- Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine** toggle is enabled on the [Playground](https://semgrep.dev/playground/new) page.
```

File: docs/release-notes/march-2023.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ This section of release notes include upgrades of Semgrep OSS Engine for version
 
 ### Added
 
-- **Semgrep Pro Engine beta** toggle is enabled by default in the [Semgrep Editor](https://semgrep.dev/orgs/-/editor/) and [Semgrep Playground](https://semgrep.dev/playground). Rules can still run with the Semgrep OSS Engine if `interfile: true` is not specified in the rule.
+- **Pro Engine beta** toggle is enabled by default in the [Semgrep Editor](https://semgrep.dev/orgs/-/editor/) and [Semgrep Playground](https://semgrep.dev/playground). Rules can still run with the Semgrep OSS Engine if `interfile: true` is not specified in the rule.
```
Comment (armchairlinguist) [inline docs/release-notes/march-2023.md:95]:
```suggestion
- **Pro Engine** toggle is enabled by default in the [Semgrep Editor](https://semgrep.dev/orgs/-/editor/) and [Semgrep Playground](https://semgrep.dev/playground). Rules can still run with the Semgrep OSS Engine if `interfile: true` is not specified in the rule.
```

**Review by armchairlinguist — COMMENTED** [review]
Need to remove beta everywhere :)

File: docs/semgrep-code/semgrep-pro-engine-intro.md (hunk 1 above)
Comment (adamkvitek) [reply docs/semgrep-code/semgrep-pro-engine-intro.md:139]: @armchairlinguist I still see "beta" in Playground:
Screenshot 2023-06-20 at 19 35 00

File: docs/release-notes/march-2023.md (hunk 4 above)
Comment (armchairlinguist) [reply docs/release-notes/march-2023.md:95]: Turns out it should be the other way around, my bad!

File: docs/semgrep-code/semgrep-pro-engine-intro.md (hunk 1 above)
Comment (armchairlinguist) [reply docs/semgrep-code/semgrep-pro-engine-intro.md:139]: Yep - turns out it should be the other way around (beta everywhere), my bad.

**Pull request merged**